### ISSUE TITLE
Menu to Tools>Multi Scene

### ISSUE THREAD

**Issue by mgmhunt** [HenrysHouses/MultiSceneTools#11]
Hi thanks for this.  I've just edited my copy to move the menu item under Tools which I think is more appropriate for this package (ie it's not 'first class Unity functionality). And the Scene Manager panel does the work.  I also use the Project Favourites to find Collections so menu items aren't really used much at all.

Thanks again. (finding a few bugs and I'm a little suspicious it's causing a crash to desktop occasionally! but early days - can't find actual evidence for this at the mo ;)

**Comment by HenrysHouses** [comment]
I'll have a look into this. and thanks for using it. There havent been to many updates here over the summer but i assure you i still working on this package. I appreciate any kind of feedback

**Comment by mgmhunt** [comment]
Just had a a thought on this - another option maybe to add the Collections that have been setup to the menu so you can quickly flip between different configurations ie BootLoader only, World Edit scene, UI editing etc etc.   Then I think having the menu at top level would be useful (more configurable the better though :)

**Comment by HenrysHouses** [comment]
for now i moved them into `Tools/Multi Scene Tools/...` but as you said maybe in the future there could be an option for users to add common collections to the menu bar for easy access 🤔

**Comment by HenrysHouses** [comment]
In inspiration to your comment i've added a menu item under `Tools/Multi Scene Tools/Add A Collection Menu Shortcut`. this automatically creates a menu item which loads the collection that is currently open. -> `Shortcuts/Load [CurrentlyLoaded] Collection` so as an example you could create shortcuts to be `Shortcuts/Load Main Menu Collection` or `Shortcuts/Load World Edit Collection`

**Comment by mgmhunt** [comment]
Awesome - I use World streamer package which means I've got to prepare scenes for Build and swap between things all the time.  Avoiding navigating the Project view all the time will add up in time savings :)

I've made a few tweaks to code for my own purposes:
- Custom Path for menu
- Added priority for menu items
- Removed Collection from Menu item and based it purely on collection name (Scene Collections in menu instead, perhaps an optional suffix could be added)
- Null check for no scene collection
- Stripped out underscores from Collection Name as I use that in Project view to sort them, but in Menu Item the underscore doesn't work, also for class name I think it'll causes problems for other characters like space

https://gist.github.com/mgmhunt/b5ac4ae093ac3276d34cc54d1b773e2c

Think ideally some of these options would exist in the manager of course. And then there's the issue when the Scene Collection name changes but eh shortcut path doesn't update.  But that's all dependant on how 'general' you want to make this.

Anyway yep sped up my workflow so thanks for that.
